app/models/search.rb

```diff
@@ -7,31 +7,59 @@ def self.scrape(isbn)
       return book
     end
 
+    good_reads_info = good_reads_api(isbn)
     google_info = google_api(isbn)
-    return nil if google_info == nil
+    world_cat_info = world_cat_api(isbn)
 
+    return nil if !google_info && !good_reads_info && !world_cat_info
```

Comment:
This would be clearer as 
return nil unless google_info || good_reads_info || world_cat_info